01_prepare_host.sh

```diff
@@ -119,8 +119,6 @@ function install_clusterctl() {
   curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/"${CAPIRELEASE}"/clusterctl-linux-amd64 -o clusterctl
   chmod +x ./clusterctl
   sudo mv ./clusterctl /usr/local/bin/clusterctl
-  mkdir -p home/"${USER}"/.cluster-api
-  touch home/"${USER}"/.cluster-api/version.yaml
```

Comment:
Note the missing `/` before `home`. This creates a folder on the relative path which doesn't seem to be used for anything. Later references to `.cluster-api` all use the absolute path `/home/${USER}/.cluster-api`.

Reply:
@furkatgofurov7 any idea why it was so and should we add a `/` before `home` instead of removing it?

Reply:
Please see the related [issue](https://github.com/metal3-io/metal3-dev-env/issues/470) and [fix](https://github.com/metal3-io/metal3-dev-env/pull/485) where these were added. IMO we need `.cluster-api ` part with missing `/` but `version.yaml` most probably might not be needed.

Reply:
That's weird. I haven't had any problems without this `.cluster-api` folder. :thinking:  Could it have been some strange value set for `$HOME`? I mean `clusterctl` should use `$HOME/.cluster-api`, right? Unless we set `--config` to something different.

Reply:
Could it be actually this issue https://github.com/kubernetes-sigs/cluster-api/issues/3573 that was the root cause for #470? The timing is suspicious and the error seems to be very much the same.

Reply:
Strange, how come then we were facing that reported issue? Most probably CAPI had issue and we fixed it before them and now the fix seems to be redundant.